lib/rucio/daemons/tracer/kronos.py

```diff
@@ -367,10 +361,11 @@ def run_once_kronos_file(heartbeat_handler: HeartbeatHandler, return_values: Dic
         ssl_key_file=ssl_key_file,
         ssl_cert_file=ssl_cert_file,
         timeout=sleep_time,
```

Comment:
Can you explain why timeout=sleep_time? In cms case, the sleep_time is 120 seconds. Why the connection should be timed out in such a short time?   We have messages to be consumed all the time, should the connection last for hours?